### ISSUE TITLE
Issue raising error when logging in

### ISSUE THREAD

**Issue by tommygod3** [bird-house/pyramid-phoenix#197]
The exception caught by: 
https://github.com/bird-house/pyramid-phoenix/blob/5c5a4f73778c31abdee13ce004dd651b43f24247/phoenix/account/base.py#L171
doesn't always have a .message attribute. Using `str(result.error)` works will all exceptions.

**Comment by tommygod3** [comment]
Error goes from:
<img width="533" alt="1" src="https://user-images.githubusercontent.com/32672006/60024450-b1dc3800-968f-11e9-91c5-cb1da4222c3a.PNG">
where the true error is obscured, to:
<img width="402" alt="2" src="https://user-images.githubusercontent.com/32672006/60024485-bbfe3680-968f-11e9-9675-ac9e7caa9951.PNG">
where the true error can be seen.

**Comment by tommygod3** [comment]
The error that can be seen in the second image seems to be a problem with the authomatic library by the way, not with Phoenix.

**Comment by cehbrecht** [comment]
Since `authomatic` is not so well maintained we might skip it and move to [oauthlib](https://github.com/oauthlib/oauthlib) only, maybe together with [pyramid-oauthlib](https://github.com/tilgovi/pyramid-oauthlib). We would have logins for CEDA/ESGF, github, google, ...

**Comment by tommygod3** [comment]
Sounds like a good solution to me.

**Comment by cehbrecht** [comment]
would you like to give it a try?

**Comment by tommygod3** [comment]
Yeah I can give it a try, removing `authomatic` and consequently OpenID, and instead using `oauthlib` sounds good. I will develop these changes in [cedadev/pyramid-phoenix](https://github.com/cedadev/pyramid-phoenix) so I can collaborate with @watucker

**Comment by tommygod3** [comment]
@cehbrecht Unfortunately my last day on placement is tomorrow and I never had the time to look at doing this
